FoxHound.Client/src/Comments/Comment.tsx

```diff
@@ -1,42 +1,37 @@
 import React from "react";
```

Comment:
This whole Comment.tsx page was mostly updated due to Prettier formatting it. The only real change here is the date view stuff because it wasn't working properly.